addons/website/static/src/client_actions/website_preview/website_builder_action.js

```diff
@@ -445,31 +446,47 @@ export class WebsiteBuilderClientAction extends Component {
                 // another page.)
                 ev.preventDefault();
             }
+
```

Comment:
To simply the code.
```js
if (this.state.isEditing) { 
                // When in edit mode, prevent the default behaviours of clicks
                // as to avoid DOM changes not handled by the editor.
                // (Such as clicking on a link that triggers navigating to
                // another page.)
                ev.preventDefault();
                return;
}

// Forward clicks to close backend client action's navbar
// dropdowns.
this.websiteContent.el.dispatchEvent(new MouseEvent("click", ev));
```
You can remove the other check `this.state.isEditing`